test/run_regression_tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e
+
+exe=../j-trove.x
```

Comment:
Very minor, possibly excessive: could we put the filename (not path) in a variable so we don't also need to declare it in  `run_benchmark.sh`?

Reply:
Do you mean in an environment variable?